site/cypress/e2e/Cypress-Feature/test_forum.spec.js

```diff
@@ -29,6 +29,36 @@ const replyToThread = (title, reply) => {
     cy.get('#posts_list').should('contain', reply);
 };
 
+const upduckPost = (thread_title) => {
+    cy.get('.thread-left-cont > .thread-list-item').contains(thread_title).click();
+    cy.wait(1000);
```

Comment:
Instead of using cy.wait, assert than an element exists, or that it contains a given value. Using explicit waits adds unnecessary time, and can lead to flaky tests later on. Apply throughout.